recognizers/pom.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.sonarsource.analyzer-commons</groupId>
+    <artifactId>sonar-analyzer-commons-parent</artifactId>
+    <version>1.5-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>sonar-analyzer-recognizers</artifactId>
+  <name>SonarQube Analyzer Recognizers</name>
+  <description>Shared code recognizers</description>
+
+  <dependencies>
+    <!-- COMPILE -->
+    <dependency>
+      <groupId>com.google.code.findbugs</groupId>
+      <artifactId>jsr305</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>commons-lang</groupId>
```

Comment:
I'm not sure about this dependency (especially regarding its scope).

Reply:
I think we need it because of [ContainsDetector](https://github.com/SonarSource/sonar-analyzer-commons/pull/40/files#diff-f038011876aeb5159363030078e05429R24) and with a "compile" scope. This declaration looks ok.